root/etc/e-smith/templates/etc/ssh/sshd_config/70SSL

```diff
@@ -0,0 +1,17 @@
+{
+    my $tlsPolicy = $tls{'policy'} || 'legacy';
+
+    if ($tlsPolicy eq 'legacy') {
+        return '';
```

Comment:
`return` statement or using `$OUT` variable? Both are good in this case, but should be used consistently.